app/src/common/gecko/com/igalia/wolvic/browser/api/impl/GeckoViewFetchClient.kt

```diff
@@ -23,8 +23,8 @@ import java.util.concurrent.TimeoutException
  * GeckoView ([GeckoWebExecutor]) based implementation of [Client].
  */
 class GeckoViewFetchClient(
-    context: Context,
-    private val maxReadTimeOut: Pair<Long, TimeUnit> = Pair(MAX_READ_TIMEOUT_MINUTES, TimeUnit.MINUTES)
+        val context: Context,
+        private val maxReadTimeOut: Pair<Long, TimeUnit> = Pair(MAX_READ_TIMEOUT_MINUTES, TimeUnit.MINUTES)
```

Comment:
Please do not include this change. You're already removing the parameter in another commit. Since you mention the `Parameter 'context' is never used` warning in this changelog I think it's better to move the removal to this commit.

Reply:
Ah, OK, I will do it and link the issue to the comments tonight.